events/events_test.go

```diff
@@ -0,0 +1,89 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package events
+
+import (
+	"testing"
+
+	"github.com/containerd/containerd/protobuf/types"
+)
+
+func TestEnvelope_Field(t *testing.T) {
+	e := &Envelope{
+		Namespace: "my-namespace",
+		Topic:     "my-topic",
+		Event: &types.Any{
+			TypeUrl: "foobar",
+			Value:   []byte("hello world"),
+		},
+	}
+
+	tests := []struct {
+		name      string
+		fieldpath []string
+		want      string
+		want1     bool
```

Comment:
nit: use `ok` or something more descriptive than `want1`

```suggestion
		ok     bool
```

Reply:
Updated, thanks for your review